lib/src/features/wine_tasting_create_view/components/characteristics/wine_characteristics_section.dart

```diff
@@ -48,98 +50,230 @@ class CharacteristicsSection extends StatefulWidget {
 class _CharacteristicsSectionState extends State<CharacteristicsSection> {
   @override
   Widget build(BuildContext context) {
-    var wineTastingState = context.watch<WineTastingCreateBloc>().state.tasting;
-
-    var characteristics = [
-      {
-        'name': 'Acidity',
```

Comment:
Add TODO for pr to remove the events related to the legacy fixed characteristics